backend/src/apiserver/server/run_server.go

```diff
@@ -66,6 +69,21 @@ func (s *RunServer) ListRuns(ctx context.Context, request *api.ListRunsRequest)
 	if err != nil {
 		return nil, util.Wrap(err, "Validating filter failed.")
 	}
+
+	if common.IsMultiUserMode() {
+		refKey := filterContext.ReferenceKey
+		if refKey != nil && refKey.Type == common.Namespace {
+			namespace := refKey.ID
+			if len(namespace) == 0 {
+				return nil, util.NewInvalidInputError("Invalid resource references for experiment. Namespace is empty.")
```

Comment:
nit: invalid resource reference, namespace is empty?

Nothing relates to experiment here, is that right?

Reply:
Good catch! Fixed.